parsers/parsers.go

```diff
@@ -1279,9 +1291,9 @@ func ParsePkGoBin(f *os.File) (*types.Pk, error) {
 	if err != nil {
 		return nil, err
 	}
-	pk.C = append(pk.C, z)
-	pk.C = append(pk.C, z)
-	pk.C = append(pk.C, z)
+	for i := 0; i < pk.NPublic+1; i++ {
```

Comment:
This also should be fixed in the `ParsePkBin` function: https://github.com/iden3/go-circom-prover-verifier/blob/feature/bugfix/parsers/parsers.go#L818

Reply:
I see the last force-push, perfect!